TLM/TLM/State/Flags.cs

```diff
@@ -732,6 +733,23 @@ public class Flags {
             }
         }
 
+        /// <summary>
+        /// removes the custom lane arrow flags. requires post recalculation.
+        /// </summary>
+        /// <param name="laneId"></param>
+        /// <returns><c>true</c>on success, <c>false</c> otherwise</returns>
+        public static bool resetLaneArrowFlags(uint laneId) {
+#if DEBUGFLAGS
+            Log._Debug($"Flags.resetLaneArrowFlags: Resetting lane arrows of lane {laneId}.");
+#endif
+            if (LaneConnectionManager.Instance.HasConnections(laneId)) {
+                return false;
+            }
+
+            laneArrowFlags[laneId] = null;
+            return true;
+        }
+
         public static bool setLaneArrowFlags(uint laneId,
```

Comment:
Naming style again. Not your code I know, but just as an example, entire module here starting functions with lowercase is not right.